src/pytest_plugins/forks/forks.py

```diff
@@ -556,9 +556,9 @@ def pytest_report_header(config, start_path):
 
 
 @pytest.fixture(autouse=True)
-def fork(request: pytest.FixtureRequest) -> Fork:
-    """Specific fork that is used to fill the test."""
-    raise Exception(f"Test '{request.node.nodeid}' was not correctly parametrized")
+def fork(request):
```

Comment:
Just curious, was the typing here causing issues? Did it get wiped out on accident but not being picked up by lint? 🤔

Reply:
It's the `raise` part: Static tests are very weird and we cannot really parametrize them, so a bunch of tricks are used to make them work, including a fake parametrization and therefore the fixture `fork` is not really overloaded like in normal tests.